lidar_pose : [0.2, 0.0]
sampling_step : 2
max_human_vel : 2.0
min_human_vel : 0.2
max_human_radius : 0.2
human_noise : 0.3
detect_range_time : 0.3
velocity_weight_prediction : 1.5
position_weight_prediction : 1.7
similar_object : 0.2
no_moving_range : 0.6
robot_base : "base_footprint"
map : "map"
removal_scan_topic : "/dr_spaam_navigation/scan"
stepping_point_topic : "/dr_spaam_navigation/object_pointers_step"
dr_spaam_topic : "/dr_spaam_detections"
merge_object_topic : "/merge_object_topic"
removal_scan_marker : "/dr_spaam_navigation/lidar_points"
step1_marker : "/dr_spaam_navigation/point_step1"
step2_marker : "/dr_spaam_navigation/point_step2"
step3_marker : "/dr_spaam_navigation/point_step3"
next_step_marker : "/dr_spaam_navigation/point_nextstep"
cost_topic : "/dr_spaam_navigation/point_cost"